examples/pose_graph/pose_graph_synthetic.py

```diff
@@ -121,8 +121,8 @@ def run(cfg: omegaconf.OmegaConf):
     dtype = torch.float64
     pg, _ = theg.PoseGraphDataset.generate_synthetic_3D(
         num_poses=cfg.num_poses,
-        rotation_noise=cfg.rotation_noise,
         translation_noise=cfg.translation_noise,
+        rotation_noise=cfg.rotation_noise,
```

Comment:
Add comment to the top of this file from #482 

`# SE(3) convention in this example is translation then rotation.`

Reply:
Added.